bin/run_first_boot_tests.py

```diff
@@ -125,6 +127,7 @@ def cat_first_boot_logs():
 
 
 if __name__ == '__main__':
+    Path(FILE_DISABLE_LAN_SERVER).touch()
```

Comment:
There's a possibility that the VM booted and started the DHCP server before this script got to create the file.
Can't we add the file from the preparation script? Do we ever want this DHCP server on if we're running in a x86 VM?